vision_datasets/common/util.py

```diff
@@ -79,3 +83,19 @@ def close(self):
     @staticmethod
     def _encode_non_ascii(s):
         return ''.join([c if ord(c) < 128 else quote(c) for c in s])
+
+
+def save_image_matching_quality(img, fp):
```

Comment:
@shonohs this roughly kept the original image quality

Reply:
nice!